src/components/icon/Icon.jsx

```diff
@@ -15,6 +15,7 @@ function Icon({
 }) {
   const classNames = cx(circle ? {} : className, {
     Icon: true,
+    'fa-sm': size === 'sm',
```

Comment:
The smaller icon better matches the design spec

Reply:
Makes sense. This isn't really a problem, but I do think it's a good opportunity to mention something about how we generally approach components:

Since we try to adhere more or less to "atomic design" principles (or just, generally, consistent design...), whenever we see an existing standard component whose props don't seem to match some designs we got, we first push back a bit on the designs to make sure it's intentional.

Specifically, since there are a lot of icons throughout the app, and so far the existing sizes have been sufficient, it means that if the designs for this one are different, then they must be inconsistent with the designs for all existing icons (we even already have several chevrons in the app being used for this exact same purpose).
At that point, the first thing we would typically do is ask the designer whether they intentionally made this icon special for some reason.

If they did, then adding a prop is absolutely correct.
If they didn't, there are 2 options:
1. The existing sizes are all a bit off from the designs
2. The designs in this case accidentally gave it the wrong size

If it's option 1, and the default-sized icons are all a bit too big, we should fix that, and make the default a bit smaller.
If it's option 2, this this icon can just be made larger.

In this case, not passing a size at all results in a height of 13.6, and sm is 11.2.
The designs have a height of 13.
So, the default size is actually closer to the designs than sm.


**TL;DR**: Again, I don't think this case is actually a problem, but I do think it's important in general to maintain consistency.
In this case, personally I probably would revert the change to Icon.jsx and go with the default size, but if you think it looks better this way, it doesn't matter much to me.